### PR TITLE
Elin Diczfalusy v5 Guess which pokemon

### PR DESCRIPTION
**Pull request by Ajliin**
All comments in README file

### CODE REVIEW

**Review by katiewu1 — COMMENTED** [review]
Cool! Really nice game with Pokemon's (learned a few 😉 ).

It's easy to follow and read the code 👍  Don't have some big suggestion of improvement. Just added a few small comments to clean the code more (it's a never ending task... so feel free if you want to do so) 😸 

Good job! 🙌 🦊

File: code/index.html [hunk 1]
```diff
@@ -3,62 +3,95 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
+    <title>Gissa pokemon?</title>
     <link rel="stylesheet" href="style.css" />
```
[the remark this answers is not in the document]
Comment (Ajliin) [reply code/index.html:7]: yes, thanks

File: code/script.js [hunk 2]
```diff
@@ -229,97 +236,181 @@ const setSecret = () => {
...
-  // Invoke a function to redraw the board with the remaining people.
-}
 
 // when clicking guess, the player first have to confirm that they want to make a guess.
 const guess = (personToConfirm) => {
-  // store the interaction from the player in a variable.
+
+window.confirm(`Vill du verkligen gissa på ${personToConfirm}`)
+  if(window.confirm){
+    personToCheck = personToConfirm
```
[the remark this answers is not in the document]
Comment (Ajliin) [reply code/script.js:365]: True, thanks

File: code/script.js [hunk 3]
```diff
@@ -229,97 +236,181 @@ const setSecret = () => {
...
-  // 4. Hide the game board
 }
 
+
 // Invokes the start function when website is loaded
 start()
 
 // All the event listeners
 restartButton.addEventListener('click', start)
+findOutButton.addEventListener('click', () => {
```
[the remark this answers is not in the document]
Comment (Ajliin) [reply code/script.js:414]: Also true, I had a console log there before

**Pull request closed without merging**